src/Surfnet/ServiceProviderDashboard/Application/Command/Entity/SaveOidcngResourceServerEntityCommand.php

```diff
@@ -373,14 +351,14 @@ public function getSupportContact(): ?Contact
     /**
      * @param Contact $supportContact
      */
-    public function setSupportContact($supportContact)
+    public function setSupportContact($supportContact): void
     {
         $this->supportContact = $supportContact;
     }
 
      /**
-     * @return string
-     */
+      * @return string
+      */
```

Comment:
Redundant and incorrect PHPDoc, fit for :wastebasket:

Reply:
Removed most of them... unfortunatly, the typing is not consistent enough to trust the phpdoc to apply it to the vars. Therefor, a lot of vars arent' still typed.

I tried some, but ran into regression errors